src/wp-includes/admin-bar.php

```diff
@@ -1253,6 +1253,39 @@ function _admin_bar_bump_cb() {
 	<?php
 }
 
+/**
+ * Remove the `_admin_bar_bump_cb` and `wp_admin_bar_header` actions
+ * and use `wp_add_inline_style` function to render inline styles.
+ *
+ * @since 6.4.0
+ */
+function wp_enqueue_admin_bar_styles() {
```

Comment:
@westonruter Based on your idea, unhook if there the action is in use and replace with `wp_add_inline_style`. If only unhooked, by third party code, do nothing.